### PR TITLE
create form to register credit card when user register account

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #262

### CODE REVIEW

File: photo-app/app/models/payment.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Payment < ApplicationRecord
+  attr_accessor :card_number, :card_cvv, :card_expires_month, :card_expires_year
+  belongs_to :user
+
+  def self.month_options
+    Date::MONTHNAMES.compact.each_with_index.map { |name, i| ["#{i+1} - #{name}", i+1]}
```
Comment (tlkiong) [inline photo-app/app/models/payment.rb:6]: Don't use single character as variable names.
Why?

Comment (LeeRenJie) [reply photo-app/app/models/payment.rb:6]: Hard to know what it is if the code base becomes larger?

Comment (tlkiong) [reply photo-app/app/models/payment.rb]: that is one. secondly, if you want to edit that variable, you can't easily select it. thirdly, with a proper name for the variable, everyone can easily understand what that variable means

Comment (LeeRenJie) [reply photo-app/app/models/payment.rb]: i have changed the variable name from "name" to "month" and "i" to "index"

File: photo-app/app/models/payment.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class Payment < ApplicationRecord
   belongs_to :user
 
   def self.choose_month
```
Comment (tlkiong) [inline photo-app/app/models/payment.rb:5]: don't understand this method name and what it does.
From the method name, it seems to be "give me the month". But, there is no input and it seems to be remapping the month data to index?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: photo-app/app/models/payment.rb (hunk 2 above)
Comment (tlkiong) [reply photo-app/app/models/payment.rb:5]: `card_expiry_month` is way better than what you have now

**Review by tlkiong — APPROVED** [review]

**Pull request merged**